INSTALL.md

```diff
@@ -20,16 +20,22 @@ conda env create --file cosmos-transfer1.yaml
 conda activate cosmos-transfer1
 # Install the dependencies.
 pip install -r requirements.txt
+# Install vllm
```

Comment:
Can we please also change "Cosmos requires the Python version to be `3.10.x`" to 3.12?